ui/src/router/components/form/components/ensembler_config/typeOptions.js

```diff
@@ -14,13 +14,26 @@ const typeOptions = [
   {
     value: "standard",
     inputDisplay: "Standard",
+    expenginetype: "standard",
```

Comment:
nit: camelCase here so `expEngineType`

Reply:
Hmm the reason why I left this as entirely lowercase was because the items in `typeOptions` do not get the `expEngineType` field stripped off when `typeOptions` gets passed into `EuiSuperSelect` as a prop, since `expEngineType` is a completely custom field not recognised by EUI. What then happens is that these items get rendered as DOM elements, with the `expEngineType` field being treated as a prop and causing this warning: 

![Screenshot 2022-09-23 at 11 17 18 AM](https://user-images.githubusercontent.com/36802364/191886630-fe52cd1b-df71-430a-bce9-ef4415cd1b74.png)

An alternative to avoiding this is to consciously strip the `typeOptions` of the `expEngineType` field every single time they are being used but I find this harder to maintain and to enforce.

Reply:
I see, thanks for clarification! The current approach is cleaner than consciously stripping the `typeOptions` value, so let's keep it this way.